working-groups/wg-replace-loader-with-lightning.md

```diff
@@ -0,0 +1,26 @@
+# Replace DM Loader with Lightning Working Group
+
+## Goals
+
+Replace DM Loader with Lightning.
+Issues:
+
+- [Replace Loader with Lightning](https://github.com/pingcap/dm/issues/405)
+
+## Meetings
+
+To be determined.
```

Comment:
please add the public meeting notes and zoom url.

Reply:
We don't really have this yet since the project is supposed to be finished in 2 weeks.

Reply:
OK. How about remove this section then? Or keep it and leave a message to explain why we don't need Meetings.

Reply:
I suggest that you could upload a meeting note or discussion records. Some files to show the process to the community would be better.